cmd/dfdaemon/app/root.go

```diff
@@ -54,25 +81,40 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-	opt.AddFlags(rootCmd.Flags())
-}
+	self, err := filepath.Abs(os.Args[0])
+	exitOnError(err, "get exec path")
+	defaultDfgetPath := filepath.Join(filepath.Dir(self), "dfget")
 
-// initOption do some initialization for running dfdaemon.
-func initOption(opt *options.Options) {
-	if opt.DfPath == "" {
-		if path, err := exec.LookPath(os.Args[0]); err == nil {
-			if absPath, err := filepath.Abs(path); err == nil {
-				// assume the dfget binary is at the same directory as this daemon.
-				opt.DfPath = filepath.Dir(absPath) + "/dfget"
-			}
-		}
-	}
+	rootCmd.Flags().BoolP("version", "v", false, "version")
```

Comment:
Use a variable to point to  rootCmd.Flags()?

Reply:
Using a variable surely make the code shorter. Fixed

Reply:
The function of `rootCmd.Flags()` will return as a `*flag.FlagSet`. So it's ok to use `rf`, but maybe `flagSet` is better(that's what we usually use). WDYT?

Reply:
I think the point of using a variable here is to make the code shorter. The scope of the variable is very small which fits into one screen, and the usage is simple and consistent, so `rf` should be clear enough.

Naming a `*flag.FlagSet` instance `flagSet` is not very meaningful since there's no additional information in the name, like `var integer int`. `rootFlags`, `rootFlagSet` and `rootCmdFlagSet` may be better, but are not as short.